src/TrafficLight.js

```diff
@@ -6,11 +6,12 @@ export default function TrafficLight(props) {
     <div className='traffic-light'>
       {/* in each of these divs, set it up so that if the color matches, add the css class 'on' to the classNames */}
       {/* for example, the css classes for the div below should be `red circle on`, if props.color is 'red' */}
-      <div className={`red circle`}></div>
+      <div className={props.color === 'red' ? 'red circle on' : 'red circle' }>
```

Comment:
This works - but wanted to note, you can also de-structure the color property from your props on line 3 (like TrafficLight({ color })) and instead of referring to it like props.color, you can just call it color. Great job!